app/audit/models/ConfirmClientDetailsAuditModel.scala

```diff
@@ -32,7 +36,9 @@ case class ConfirmClientDetailsAuditModel(clientName: String, nino: String,
     "mtditid" -> mtditid,
     "saUtr" -> saUtr,
     "agentReferenceNumber" -> arn,
-    "userType" -> "Agent") ++
+    "userType" -> "Agent",
```

Comment:
worth to bundle thins into one single field, but depends on the requirements I guess